share/functions/__fish_config_interactive.fish

```diff
@@ -338,4 +337,11 @@ function __fish_config_interactive -d "Initializations that should be performed
             end
         end
     end
+
+    set __fish_initialized $initializing
+    # Remove some deprecated variables.
```

Comment:
I'd recommend against this, as removing these makes downgrading a pain.

Just keep them around, they cost nothing.

Reply:
Yeah, done, I hadn't thought about downgrading